jigsaw-game/client/menu.py

```diff
@@ -0,0 +1,110 @@
+import pygame
+from pygame.locals import QUIT, MOUSEBUTTONDOWN
+
+pygame.init()
+
+# Colors
+WHITE = (255, 255, 255)
+RED = (255, 0, 0)
+BLUE = (0, 0, 255)
+
+# Screen dimensions
+WIDTH, HEIGHT = 800, 600
+screen = pygame.display.set_mode((WIDTH, HEIGHT))
+pygame.display.set_caption('Game Menu')
+
+font = pygame.font.SysFont(None, 36)  # Default font, size 36
```

Comment:
this should be in `menu` function and dimensions could be put into a `constants.py` file as `GameClient` uses it aswell, but we should resizeable, and this should probably a "configurable" option in "Settings".. do have fun with fonts